conans/client/conf/config_installer.py

```diff
@@ -201,9 +207,12 @@ def _process_config(config, cache, output, requester):
             _process_git_repo(config, cache, output)
         elif config.type == "dir":
             _process_folder(config, config.uri, cache, output)
-        elif config.type == "file":
+        elif config.type == "compressed":
             with tmp_config_install_folder(cache) as tmp_folder:
                 _process_zip_file(config, config.uri, cache, output, tmp_folder)
+        elif config.type == "file":
```

Comment:
Hi @sagarafr

I am having a look, and there is a problem: we cannot change existing behavior unless it is declared a bug. At the moment the type for a zipped file, the type is "file", and that cannot be changed. It is not good enough to introduce a ``compressed`` type for differentiating this. The current ``file`` behavior has to be respected and maintained.

So I think it would be better to try to process files capturing the exception when the decompression fails.

Reply:
Indeed, I understand my mistake when I have push. I just try to find the right place to make the check in order to have a cleaner code and have all tests pass.